docs/DeveloperGuide.md

```diff
@@ -154,6 +154,117 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Manual Command
+
+#### Description
+
+The `manual` command displays the format for a specified command and a short description for a particular command. Since the field is not compulsory during the execution of the `manual` command, the user's input is being parsed in `ManualCommandParser`. After which, a new `ManualCommand` object will be created, and is executed by the `LogicManager`.
+
+#### Implementation
+1. Upon receiving the user input, the `LogicManager` starts to parse the given input text using `AddressBookParser#parseCommand()`.
+2. The `AddressBookParser` invokes the respective `Parser` based on the first word of the input text.
+3. Since the first word in the user input matches the word "manual", `AddressBookParser#parseCommand(arguments)` will be called. In this case, the arguments refer to the remaining input text after the exclusion of the command word "manual".
```

Comment:
```suggestion
3. Since the first word in the user input matches the word "manual", `ManualCommandParser#parse(arguments)` will be called. In this case, the arguments refer to the remaining input text after the exclusion of the command word "manual".
```